docs/google-cloud.md

```diff
@@ -34,107 +44,121 @@ gcloud auth configure-docker
 PATH="$PATH:$(pwd)/bin"
 ```
 
-### And run the configuration script
+### Run the configuration script
 
 ```
 kctf-config-create
 ```
 
-### Configuration properties
-Type a path for storing the challenges
+### Set configuration properties
+Enter a path for storing the challenges:
 ```
 ~/demo-ctf-cluster
 ```
 
-Type your project id:
+Enter your project id:
 ```
-{{project-id}}
+{{ project-id }}
 ```
 
-You can use the default values for all other settings.
+### Other settings 
+For all other settings, use the default values.
+
+## Create the cluster
+After configuring the project, the cluster is created automatically.
 
-## And finally, create the cluster
-After this is done, the cluster will be created.
+This is only done once per CTF. A "cluster" is essentially a group of VMs with a "master" that defines what to run there.
 
-This only needs to be done once per CTF. A "cluster" is essentially a group of VMs with a "master" that defines what to run there.
+It can take around a minute for the cluster to be created, during which the following message is displayed:
 
-This can take a while, you can expect to see `Creating cluster ... in ... Cluster is being health-checked...` and in a minute or so, you'll get a message telling you it's done.
+```Creating cluster ... in ... Cluster is being health-checked...``` 
+
+You'll get a message when cluster creation has completed.
 
 While you wait, here's how the infrastructure works:
-1. The CTF challenges will run inside "nsjail" (a security sandbox).
-1. The contents of the nsjail sandbox will be configured in a docker container.
-1. The container will be deployed using Kubernetes in a group of VMs.
+1. The CTF challenges run inside "nsjail" (a security sandbox).
+1. The contents of the nsjail sandbox are configured in a docker container.
+1. The container is deployed using Kubernetes in a group of VMs.
 1. If the VMs consume too much CPU, Kubernetes automatically deploys more VMs.
-1. If the VMs consume too little CPU, Kubernetess will shut down some VMs.
-1. Some very expensive challenges will probably be allocated in the same VMs as less busy challenges.
-
-This all ensures the availability of the challenges, and saves some computing resources.
+1. If the VMs consume too little CPU, Kubernetes will shut down some VMs.
+1. Some very expensive challenges can be allocated to the same VMs as less busy challenges.
 
-Anyway, hopefully by now the cluster is already created and we can continue with the walkthrough.
+The above steps ensure the availability of the challenges, while using computing resources in a resourceful manner.
 
-If you are curious and have some more time, take a look at this [kCTF introduction](introduction.md), which includes a quick 8 minutes summary of what is kCTF and how it interacts with Kubernetes.
+By now, your cluster should be ready and we can continue with the walkthrough.
```

Comment:
unfortunately, this is probably not gonna be true. this can take several minutes :(

Reply:
see comment above for ideas

Reply:
we could rephrase: "Your cluster should soon be ready, when it is, you can continue with this walkthrough."

Reply:
SGTM! lets do that :)